contrib/checkbox-provider-ce-oem/bin/serial_console_test.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+import serial
+import serial.rs485
+import logging
+import sys
+import os
+from argparse import ArgumentParser
+
+formatter = logging.Formatter(
+    "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+)
+handler = logging.StreamHandler(sys.stdout)
+handler.setFormatter(formatter)
+logger = logging.getLogger("serial-test")
+logger.addHandler(handler)
+logger.setLevel(logging.INFO)
+
+
+def test_serial_console(serial_device, baudrate):
+    """Test the serial port when it is in console mode"""
+    try:
+        # Open serial port
+        with serial.Serial(serial_device, baudrate, timeout=1) as ser:
+            logger.info("Serial port opened successfully.")
+
+            # Send 'Enter Key'
+            logger.info("Sending 'Enter Key'...")
+            ser.write(os.linesep.encode())
+            response = ser.read(size=128).decode()
+            logger.info("Received response: '%s'", response)
+            # ":~$" is the pattern for the DUT after logging in
+            # "login:" is the pattern for the DUT before logging in
+            if ":~$" in response or "login:" in response:
+                logger.info("[PASS] Serial console test successful.")
+            else:
+                logger.info("[FAIL] Serial console test failed.")
+                logger.info(
```

Comment:
`logger.info` -> `logger.error`

Reply:
I think `error` is for the time when code is not running in the right way such as any unexpected behavior. In this case, program run without error, so it should be `info`
What do you say?